### DISCUSSION TITLE
Add possibility to add specific column to Table Dashboard based on specific attribute

### DISCUSSION THREAD

**Discussion by AMontagu** [grafana/grafana#67816] [Dashboards, Panels & Visualizations]
Hello

Context:
- Span name from opentelemetry js is low in cardinality and use HTTP {METHOD_NAME} as span name instead of URL used. (See https://github.com/open-telemetry/opentelemetry-js/issues/2882 never finished)
- Table from grafana dashboard display "Trace ID", "Start time", "Name" and "Duration" column

Issue:
All span in table look like: HTTP GET or HTTP POST and do not bring a lot of value.
![Screenshot from 2023-05-04 11-50-56](https://user-images.githubusercontent.com/11883225/236171297-053db82b-f7d4-482f-a1d1-83f6ae586a2b.png)

Proposition:
- Add an SQL like annotate that allow to inject span attribute or ressources into the table
- Ability to add custom colum that query span attributes or ressources

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82670. Feel free to continue the discussion around this there. Thank you!
